dali/imgcodec/decoders/nvjpeg2k/nvjpeg2k_test.cc

```diff
@@ -64,21 +70,27 @@ const std::vector<std::pair<std::string, ROI>> roi_images = {
   {"2/tiled-cat-3113513_640", {{2, 1}, {200, 600}}},
 };
 
+const char bitdepth_converted_imgname[] = "0/cat-1245673_640";
+
 struct ImageTestingData {
   std::string img_path;
   std::string ref_path;
   ROI roi;
+};
 
-  explicit ImageTestingData(std::string filename)
-  : img_path(join(img_dir, filename) + ".jp2")
-  , ref_path(join(ref_dir, filename) + ".npy")
-  , roi() {}
+ImageTestingData from_regular_file(std::string filename) {
```

Comment:
Please consider std::filesystem::path

Reply:
The filesystem library unfortunately is not on the CI, experimental/filesystem could be used but I didn't bother to use it.